bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BridgeHandler.java

```diff
@@ -195,6 +203,10 @@ public void dispose() {
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
         // commands are handled by individual device handlers
+        if (channelUID.getId().equals(BoschSHCBindingConstants.CHANNEL_EXECUTE_SCENARIO)
+                && !command.toString().equals("REFRESH") && this.httpClient != null) {
+            this.scenarioHandler.executeScenario(this.httpClient, command.toString());
```

Comment:
To fix null annotation issue:
```suggestion
        BoschHttpClient httpClient = this.httpClient;
        if (channelUID.getId().equals(BoschSHCBindingConstants.CHANNEL_EXECUTE_SCENARIO)
                && !command.toString().equals("REFRESH") && httpClient != null) {
            this.scenarioHandler.executeScenario(this.httpClient, command.toString());
```

This is because another thread could have set `this.httpClient = null` between your check and method call.